app/Repositories/Contracts/PostRepository.php

```diff
@@ -0,0 +1,19 @@
+<?php
+/**
+ * Created by PhpStorm.
+ * User: quantien
+ * Date: 11/4/18
+ * Time: 6:20 PM
+ */
+
+namespace App\Repositories\Contracts;
+
+
+interface PostRepository extends AbstractRepository
+{
+    public function getAll($data = []);
+
+    public function show($id);
+
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: PSR2.Files.EndFileNewline.NoneFound</br>**Severity**: error</br>**File**: app/Repositories/Contracts/PostRepository.php</br>**Line**: 19</br>Expected 1 newline at end of file; 0 found